test/generators/install_generator_test.rb

```diff
@@ -90,7 +90,7 @@ class InstallGeneratorTest < Rails::Generators::TestCase
   test "adds host config to development.rb" do
     run_generator
     assert_file "config/environments/development.rb" do |config|
-      assert_match "config.hosts = (config.hosts rescue []) << /\\h+.ngrok.io/", config
+      assert_match "config.hosts = (config.hosts rescue []) << /\\w+\\.ngrok\\.io/", config
```

Comment:
Would it be easier to read if the test was like that?
```rb
assert_match 'config.hosts = (config.hosts rescue []) << /\w+\.ngrok\.io/', config
```

Reply:
Looks the same to me, I think it's fine as is.